README.md

```diff
@@ -39,14 +42,32 @@ $ dnsbl-exporter -h
 
  Go to http://127.0.0.1:9211/ in your browser.
 
+#### Using Docker
+
+There's **no** `:latest` tag by design. But we build individual releases for amd64 and arm64.
+
+````sh
+$ export DNSBL_EXP_RESOLVER=127.0.0.1:53
```

Comment:
This shouldn't work. I should extend the example to show how to use a dockerized unbound, or use `host.docker.internal` or `docker.for.mac.host.internal `.